index.html

```diff
@@ -2,42 +2,188 @@
-        <div class="navbar-nav">
-          <a class="nav-link active ali me-2" aria-current="page" href="#">Home</a>
-          <a class="nav-link me-2" href="#">About Us</a>
-          <a class="nav-link me-2" href="#">Agents</a>
-          <a class="nav-link " href="#">Blog</a>
+    <!-- MAIN HEADER -->
+    <div class="container-fluid main-heading">
+
+        <nav class="navbar navbar-expand-lg  text-light">
+            <div class=" container">
+                <a class="navbar-brand" href="#">Berkayu</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse align-content-center nav-items " id="navbarNavAltMarkup">
+                    <div class="navbar-nav">
+                        <a class="nav-link active ali me-2" aria-current="page" href="#">Home</a>
+                        <a class="nav-link me-2" href="#">About Us</a>
+                        <a class="nav-link me-2" href="#">Agents</a>
+                        <a class="nav-link " href="#">Blog</a>
+                    </div>
+                </div>
+                <div>
+                    <button type="button" class="btn btn-primary border-radius ">Sing In</button>
+                    <button type="button" class="btn btn-light border-radius">Try It Now</button>
+                </div>
+            </div>
+        </nav>
+        <!-- MAIND HEADER CONTENT -->
+        <div class="container text-light main-heading-title ">
+            <h1>Easy way to find <br>your dream <strong>house <br>& real estate </strong> </h1>
+            <p class="mt-4">with the most complete source of homes for sale & <br> real estate for you </p>
+            <button type="button" class="btn btn-danger mt-3 border-radius">Search now</button>
+        </div>
+        <!-- CARDS -->
+    </div>
+    <div class="container-fluid bg-light text-dark">
+        <div class="row justify-content-center bg-light ">
+
+            <div class="card col-2 left-card info-cards text-center  " style="width: 25rem;">
+                <img src="img/unnamed.gif" class="card-img-top rounded-circle info-cards-image mt-4  " alt="...">
+                <div class="card-body ">
+                    <h3 class="card-title mb-3 ">i'm looking to buy a house</h3>
+                    <p class="card-text mb-3">want a comfortable home with complete facilities? find your dream home
+                        here!</p>
+                    <a href="#" class="btn left-btn border-radius px-5 py-2 mb-3">Search Now</a>
+                </div>
+            </div>
+            <!--  right card -->
+            <div class="card col-2 right-card info-cards text-center " style="width: 25rem;">
```

Comment:
when you have a CSS file, you shouldn't write inline CSS like this, it's illegal. please change it.